app/model/ingest.py

```diff
@@ -79,3 +80,20 @@ class IngestDocumentDTO(BaseModel):
     title: str
     source_url: Optional[AnyHttpUrl] = None
     user_language_name: Optional[str]
+
+    def to_document_create_dto(self) -> DocumentCreateDTO:
+        """
+        Convert IngestDocumentDTO to DocumentCreateDTO.
+
+        :return DocumentCreateDTO: Converted DocumentCreateDTO instance.
+        """
+
+        return DocumentCreateDTO(
+            import_id=self.import_id,
+            family_import_id="",
```

Comment:
Note to self - check on if allowing this is a good thing

Reply:
I've raised https://linear.app/climate-policy-radar/issue/PDCT-1420/ensure-import-ids-cannot-be-empty-strings

Reply:
Resolve if you agree :smile:

Reply:
Good idea